programs/steward/src/state/accounts.rs

```diff
@@ -15,20 +15,39 @@ pub struct Config {
     /// SPL Stake Pool address that this program is managing
     pub stake_pool: Pubkey,
 
-    /// Authority for pool stewardship, can execute SPL Staker commands and adjust Delegation parameters
-    pub authority: Pubkey,
+    /// Validator List
+    pub validator_list: Pubkey,
+
+    /// Admin
+    /// To be controlled by a committee of stakeholders
+    /// - Update the `parameters_authority`
+    /// - Update the `blacklist_authority`
+    /// - Can call SPL Passthrough functions
+    /// - Can pause/reset the state machine
+    pub admin: Pubkey,
+
+    /// Parameters Committee
+    /// To be controlled by a committee of stakeholders
+    /// - Can update steward parameters
+    pub parameters_authority: Pubkey,
+
+    /// Blacklist Committee
+    /// To be controlled by a committee of stakeholders
```

Comment:
Let's remove the "committee" language / terminology, since we don't want to be beholden to just that setup - could be dao, another program, multisig, ledger, etc

Reply:
10-4 Done